modules/sdk/project-templates/project-templates-modules-ext/build.gradle

```diff
@@ -0,0 +1,13 @@
+dependencies {
+	compile project(":sdk:project-templates:project-templates")
```

Comment:
The `modules-unit-project-templates-jdk8` CI test failed and it looks like a chicken and egg problem

```
Could not find com.liferay:com.liferay.project.templates:4.1.7.
...
Required by: project :sdk:project-templates:project-templates > com.liferay:com.liferay.project.templates.modules.ext:1.0.0
````

I think we might need to revert f4ef33cc.

Reply:
Also for the `project-templates-modules-ext` I had to leave it using a `compile project(...)` style dependency because the `ModulesExtProjectTemplatesCustomizer` is depending on API that isn't yet available in latest published project-templates.  Once we publish that one, we can switch it as well.

Reply:
For the classpath, does this work?

```
classpath = sourceSets.main.runtimeClasspath + configurations.compileOnly
```